src/Microsoft.AspNet.Razor/Parser/TagHelpers/TagHelperParseTreeRewriter.cs

```diff
@@ -648,7 +648,7 @@ private static bool IsEndTag(Block tagBlock)
 
             var childSpan = (Span)tagBlock.Children.First();
             // We grab the symbol that could be forward slash
-            var relevantSymbol = (HtmlSymbol)childSpan.Symbols.Take(2).Last();
+            var relevantSymbol = (HtmlSymbol)childSpan.Symbols[1];
```

Comment:
Doesn't do the same thing as it did before. `Take(2)` of a list of symbols of length 1 only returns a single length enumerable.